src/example/incompressible_navier_stokes/incompressible_navier_stokes.jl

```diff
@@ -0,0 +1,304 @@
+#
+# # Description of the case
+# The simulation domain consists in a channel bounded above and below by two walls and within which a cylinder is placed. The geometrical parameters of the case
+# are shown in the Figure below. 
+#
+# ![](../assets/navier_stokes_cylindre.png)
+#
+# The flow around the cylinder is governed by the incompressible Navier-Stokes equations:
+# ```math
+#   \partial_t u + u \cdot \nabla u = - \nabla p + \nu \Delta u
+# ```
+# ```math
+#   \nabla \cdot u = 0
+# ```
+# 
+# To discretize the equations $\mathbb{P}_2$-$\mathbb{P}_1$ elements are used (Taylor-Hood). 
+# 
+# No-slip boundary conditions are applied on all walls. The inlet is given by an imposed parabolic velocity profile which
+# is ramped up in time:
+# ```math
+#   u(t,x,y) = u_{in}(t) \times \left( 4  y (0.41-y)/0.41^2 , 0 \right)
+# ```
+# where $u_{in}(t) = \textrm{clamp}(t,0.0,1.5)$. 
+# 
+# At the outlet, $p=0$ is imposed when using the projection method and a "do-nothing" ($-pn + \nu \nabla u \cdot n = 0$) condition is applied when using the "mixed form" method.
+#
+# # Code
+# Load the necessary packages
+const dir = string(@__DIR__, "/") # bcube/example dir
+using Bcube
+using LinearAlgebra
+using WriteVTK
+using StaticArrays
+
+# Function space (here we shall use Taylor-Hood P2-P1 elements) and quadrature degree.
+const fspace = :Lagrange
+const degree_u = 2
+const degquad = 2 * degree_u + 1
+const degree_p = 1
+
+# Input and output paths
+const outputpath = joinpath(dir, "../../../myout/navier_stokes/")
+const meshpath = joinpath(dir, "../../../input/mesh/cylinder_navier_stokes_tri.msh")
+
+# Kinematic viscosity
+const ν = 0.001
+
+# Time step and simulation time
+const Δt = 1.0e-3
+const finalTime = 6.0
+
+# Function that defines the inlet velocity profile
+function inlet_velocity(x, t)
+    SA[4.0 * clamp(t, 0.0, 1.5) * x[2] * (0.41 - x[2]) / (0.41 * 0.41), 0.0]
+end
+
+# Function that solves the problem using the projection method
+function run_unsteady_projection_method()
+    # Read mesh
+    mesh = read_msh(meshpath)
+
+    # Definition of trial and test function spaces (with associated Dirichlet boundary conditions)
+    fsu = FunctionSpace(fspace, degree_u)
+    U_vel = TrialFESpace(
+        fsu,
+        mesh,
+        Dict(
+            "left" => inlet_velocity,
+            "top" => SA[0.0, 0.0],
+            "bottom" => SA[0.0, 0.0],
+            "cylinder" => SA[0.0, 0.0],
+        ); # 
+        size = 2,
+    )
+    V_vel = TestFESpace(U_vel)
+
+    fsp = FunctionSpace(fspace, degree_p)
+    U_pre = TrialFESpace(fsp, mesh, Dict("right" => SA[0.0]); size = 1)
+    V_pre = TestFESpace(U_pre)
+
+    velocity = FEFunction(U_vel, 0.0)
+    pressure = FEFunction(U_pre, 0.0)
+
+    # Define measures for cell
+    dΩ = Measure(CellDomain(mesh), degquad)
+
+    # Definition of bilinear and linear forms
+    # Tentative velocity forms
+    m1(u, v) = ∫(u ⋅ v)dΩ
```

Comment:
there is a `build_mass_matrix(U, V, dΩ::AbstractMeasure)` in Bcube that wraps the bilinear form + assembly, but as you prefer